adobecom/milo
# MWPW 143314: unav notifications analytics

## Description
This PR adds analytics support for the universal navigation notifications

## Related Issue
Resolves: [MWPW-143314](https://jira.corp.adobe.com/browse/MWPW-143314)

## Testing instructions
1. go to the network tab in dev tools and search for `'collect'`;
2. click through the notifications part of the universal navigation;
3. you'll find the sent tag inside the payload, under `events[0].xdm.web.webInteraction.name`, as you can see in the screenshot below;

_Sent tags can be checked against [this](https://wiki.corp.adobe.com/pages/viewpage.action?spaceKey=marketingtech&title=Universal+Nav+on+Adobe.com+-+Custom+Tracking). Support for 18, 20 is not yet provided and 21 will send the same event as 19._

<img width="1549" alt="Screenshot 2024-03-07 at 07 44 25" src="https://github.com/adobecom/milo/assets/146744221/57f4b2f9-42f9-470d-a132-74fdad212936">

## Test URLs
**Milo:**
- Before: https://main--milo--robert-bogos.hlx.page/drafts/rbogos/page-default
- After: https://mwpw-143314-unav-notifications-analytics--milo--robert-bogos.hlx.page/drafts/rbogos/unav
